content/livesync/models/models.textile

```diff
@@ -93,30 +92,26 @@ SELECT COALESCE(MAX(sequence_id), 0) FROM outbox;
 COMMIT;
 ```
 
-For this to work correctly, use a "Read Committed":https://www.postgresql.org/docs/7.2/xact-read-committed.html transaction isolation level when reading your database state. This is the default isolation level in PostgreSQL.
+Use the *read committed transaction isolation level* for the correct function, when reading the database state. "PostgreSQL uses this":https://www.postgresql.org/docs/7.2/xact-read-committed.html as default. 
```

Comment:
I think this is actually incorrect, which was my error before, sorry. We need repeatable read isolation: https://www.postgresql.org/docs/current/transaction-iso.html#XACT-REPEATABLE-READ